### PR TITLE
lives as hearts

### PR DESCRIPTION
**Pull request by topace14**
now shows hearts for lives, harlequin for widow, skulls for dead. swapped cut and table to see what it would look like

### CODE REVIEW

**Comment by qsavit** [comment]
@topace14 why not just use emoji for all these symbols?

File: src/server.js [hunk 1]
```diff
@@ -197,12 +197,11 @@ export class Server {
                     player.isActive = true;
                 } else if (!this.widowUsed){
                     this.widowUsed = true;
+                    player.numLives = 0;
                     this.playersList.getActivePlayer().isActive= false;
                     player.isActive = true;
-                    // Alter player name to make fun of them
-                    player.name = '💩' + player.name.substring(1);
```
Comment (qsavit) [inline src/server.js:203]: yesss

Comment (qsavit) [reply src/server.js:203]: why you remove??

Comment (topace14) [reply src/server.js:203]: Well, the game is going to be going pretty strongly in the pixel art direction, so the poop emoji is replaced with a pixel art jester's harlequin.

Comment (qsavit) [reply src/server.js:203]: let's split the difference: pixel art poop emoji? :D

Comment (franc0is) [reply src/server.js:203]: Lots of options :-D

image

image

image

image

**Pull request merged**